ff_stdio.c

```diff
@@ -2167,10 +2167,9 @@ int prvFFErrorToErrno( FF_Error_t xError )
 
 #if ( ffconfigTIME_SUPPORT == 1 )
 
-    static uint32_t prvFileTime( FF_SystemTime_t * pxTime )
+    time_t prvFileTime( FF_SystemTime_t * pxTime )
```

Comment:
Here above `prvFileTime()` was declared private. I don't think that the function is useful outside this module.